aea/multiplexer.py

```diff
@@ -274,7 +277,7 @@ async def disconnect(self) -> None:
         self.logger.debug("Multiplexer disconnecting...")
         async with self._lock:
             if self.connection_status.is_disconnected:
-                self.logger.debug("Multiplexer already disconnected.")
+                self.logger.debug("Multiplexer disconnected.")
```

Comment:
Seems like cheating...double disconnections will not be as easy to notice...

Reply:
fixed